config/routes.js

```diff
@@ -146,6 +146,7 @@ module.exports.routes = {
   'GET    /noo/logout':                                   'SessionController.destroy',
 
   'POST    /noo/pn/add-device':                           'PushNotificationController.addDevice',
+  'POST    /noo/pn/disable-device':                       'PushNotificationController.disableDevice',  
```

Comment:
By convention, the id (in this case the token) is only present in the URL
if you're updating an existing record. it doesn't mean you can't still pass
the token in as a query parameter or POST body parameter.

On Mon, Jul 6, 2015 at 2:51 PM, Robbie Carlton <notifications@github.com>
wrote:

> In config/routes.js
> <https://github.com/Hylozoic/hylo-node/pull/29#discussion_r33986811>:
>
> > @@ -146,6 +146,7 @@ module.exports.routes = {
> >    'GET    /noo/logout':                                   'SessionController.destroy',
> >
> >    'POST    /noo/pn/add-device':                           'PushNotificationController.addDevice',
> > +  'POST    /noo/pn/disable-device':                       'PushNotificationController.disableDevice',
>
> cool, so would it work if it were
> POST /noo/device/:token => DeviceController.create
> and
> DELETE /noo/device/:token => DeviceController.destroy
> as the create method needs the token as well?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Hylozoic/hylo-node/pull/29/files#r33986811>.
>